.github/workflows/build.yml

```diff
@@ -0,0 +1,42 @@
+name: build-manual
+run-name: Build manual
+on: [push, workflow_dispatch]
+
+permissions:
+  contents: read
+  pages: write
+  id-token: write
+
+jobs:
+  build:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v3
+      - name: Get dependencies
+        run: |
+          sudo apt-get update
+          sudo apt-get install -y texlive-latex-recommended texlive-latex-extra texlive-science
```

Comment:
Probably better to cache this, TeX can be a heavy dependency: 

https://github.com/marketplace/actions/cache-apt-packages